compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -345,6 +345,50 @@ def IREEGPU_DataTiledMMAAttr :
   );
 }
 
+def IREEGPU_DataTiledScaledMMAAttr :
+    IREEGPU_DataTiledMMAInnerTileDescAttr<"DataTiledScaledMMA", [
+  DeclareAttrInterfaceMethods<IREECodegen_InnerTileDescAttrInterface, [
+    "getExpectedNumInputs",
+    "getExpectedNumOutputs",
+    "verifyIndexingMaps",
+    "buildUnderlyingOperations"
+  ]>
+]> {
+  let mnemonic = "data_tiled_scaled_mma_layout";
+  let cppNamespace = "::mlir::iree_compiler::IREE::GPU";
+
+  let description = [{
+    This mma variant represents scaled MMA ops with data-tiling details. The
+    |intrinsic| field specifies which particular scaled MMA intrinsic is
+    targeted by the data-tiling.
+
+    The |lhs_elem_type|, |rhs_elem_type|, and |acc_elem_type| fields specify the
+    element types of the LHS, RHS, and ACC operands respectively. The element
+    types for the scales operands are always Float8E8M0FNUType, so they are not
+    specified on the attribute.
+
+    The other fields default to one, and that default results in a single
+    intrinsic, while values greater than one result in wider "kernels"
+    consisting of multiple intrinsic instructions, with the data layout already
+    swizzled into a tile layout that allows each intrinsic to access data at an
+    offset that's as simple as possible a mapping from the thread ID.
```

Comment:
I know it's somewhere else in a comment, but I think it would be useful to describe the interleaved layout for `intrinsics_k` here as well.